### ISSUE TITLE
@font-face breaks other fonts

### ISSUE THREAD

**Issue by NielsFestjens** [wkhtmltopdf/wkhtmltopdf#3368]
When I load a custom font, all other fonts are ignored and rendered by the default font.
Wkhtmltopdf version: 0.12.4 (I downloaded the lastest version).
I downloaded the 0.13.0-apha, with that version I did not have the problem.

In the sample below, the Jolly Lodger font will be rendered correctly, all the other fonts resort to the default. iI I remove the `<link>` to the stylesheet then all the other fonts get rendered as expected.

```<!DOCTYPE HTML>
<html>
	<head>
		<link href='http://fonts.googleapis.com/css?family=Jolly+Lodger' rel='stylesheet' type='text/css'>
	</head>
	<body>
		<table>
			<tr>
				<td style="font-family:'Calibri'">Calibri</td>
				<td>Calibri</td>
			</tr>
			<tr>
				<td style="font-family:'DejaVu Sans Condensed'">DejaVu</td>
				<td>DejaVu</td>
			</tr>
			<tr>
				<td style="font-family:'Consolas'">Consolas</td>
				<td>Consolas</td>
			</tr>
			<tr>
			<tr>
				<td style="font-family:'Consolas'">Arial</td>
				<td>Arial</td>
			</tr>
			<tr>
				<td style="font-family:'Times New Roman'">Times New Roman</td>
				<td>Times New Roman</td>
			</tr>
			<tr>
				<td style="font-family:'Jolly Lodger'">Jolly Lodger</td>
				<td>Jolly Lodger</td>
			</tr>
		</table>
	</body>
</html>

**Comment by PhilterPaper** [comment]
If you remove the `<link>`, and all the other fonts are correct, can I assume that Jolly Lodger uses a default font then? That would tell you that you _are_ using the remote copy of JL and not a local one. As for why putting a couple of @font-faces in the `<head>` breaks things, try putting `<style>` tags around the link. They're just bare @font-face entries, so that might be breaking something.

**Comment by NielsFestjens** [comment]
Indeed, removing the `link` results in the default font where it says `<td style="font-family:'Jolly Lodger'">Jolly Lodger</td>`.
I used Jolly Lodger because it's an easy example that I can show here, in my original case it's a custom font, and we load it through internal css, so there the `@font-face` is wrapped within a `style` tag.
As for wrapping the `<link>` in a `<style>` tag, that does not make sense, `<link>` is for external stylesheets, <style> is for internel stylesheets.

**Comment by goulashsoup** [comment]
This bug is not in Version 0.13, but if you want to use special options like `--header-html`, `--footer-html`, etc. you have to download a 0.13 version that is compiled with _patched Qt_ .

[Here](https://github.com/poizan42/wkhtmltopdf/releases/tag/0.13-qt5.6-alpha1) you can find a build with patched Qt.

[Here](https://github.com/wkhtmltopdf/wkhtmltopdf/issues/3185) is the _patched Qt_ issue.
